presto-geospatial/src/main/java/io/prestosql/plugin/geospatial/GeoFunctions.java

```diff
@@ -284,6 +300,23 @@ public static Slice stGeomFromBinary(@SqlType(VARBINARY) Slice input)
         return serialize(geomFromBinary(input));
     }
 
+    @Description("Returns a Geometry type object from Spatial Framework for Hadoop representation")
+    @ScalarFunction("geometry_from_hadoop_shape")
+    @SqlType(GEOMETRY_TYPE_NAME)
+    public static Slice geometryFromHadoopShape(@SqlType(VARBINARY) Slice input)
+    {
+        requireNonNull(input, "input is null");
+
+        try {
+            OGCGeometry geometry = OGCGeometry.fromEsriShape(getShapeByteBuffer(input));
+            String wkt = GeometryEngine.geometryToWkt(geometry.getEsriGeometry(), getWktExportFlags(input));
```

Comment:
We can use static import for `geometryToWkt` since the name is self descriptive.